mongo/options/clientoptions_1_9.go

```diff
@@ -4,10 +4,17 @@ package options
 
 import (
 	"crypto/x509"
+	"fmt"
 )
 
-// We don't support version less then 1.10, but Evergreen needs to be able to compile the driver
-// using version 1.8.
+// We don't support Go versions less than 1.10, but Evergreen needs to be able to compile the driver
+// using version 1.8 and cert.Subject
```

Comment:
Nice, thanks for improving the comments. I think this reference 1.9 as the minimum:
```suggestion
// using version 1.9 and cert.Subject
```

Reply:
Sounds good!